vocab_tools/__init__.py

```diff
@@ -21,6 +21,7 @@
     "skos": "http://www.w3.org/2004/02/skos/core#",
     "obo": "http://purl.obolibrary.org/obo/",
     "dcterm": "http://purl.org/dc/terms/",
+    "schema": "https://schema.org/",
```

Comment:
Namespace for schema.org is `http://schema.org/`

Reply:
Done.  had to change the URI in all the vocabularies too.